docker-compose.yml

```diff
@@ -13,11 +13,10 @@ services:
       - MYSQL_RANDOM_ROOT_PASSWORD=yes
 
   selenium:
-    image: selenium/standalone-firefox-debug:3.141.59-zirconium
-    volumes:
-      - /dev/shm:/dev/shm
+    image: selenium/standalone-firefox:4.25
```

Comment:
Shouldn't we just use `image: selenium/standalone-firefox:latest` ? It is just the Docker image version, not even the browser version so we might as well be as up-to-date as possible here IMO.

https://hub.docker.com/r/selenium/standalone-firefox

Reply:
Using selenium/standalone-firefox:latest could seem convenient for keeping the Docker image up to date, but it also introduces some risks. Since the browser version is tied to the Docker image, any updates in latest could unexpectedly impact our setup. If Selenium updates to a newer version, some functions might be deprecated, leading to compatibility issues or unexpected errors in our testing environment. To ensure stability, it might be better to specify a fixed image version we know works well, rather than relying on latest.

Reply:
Oh, it seems I misunderstood the documentation then. OK.